source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,121 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. The fastest way to get your first capture with the Zivid SDK!
+2. How can you configure the settings of the Zivid camera?
+3. More on captures - HDR and 2D
+4. Bonus!
+
+### Prerequisites
+
+You should have installed Zivid SDK and C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Part 1 - Quick Start
+
+### Connect
+
+First we have to connect to the camera ([go to source][connect-url]).
+```cpp
+auto camera = zivid.connectCamera();
+```
+
+### Capture
+
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```cpp
+auto frame = camera.capture();
+```
+
+### Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
+```cpp
+auto resultFile = "result.zdf";
```

Comment:
I would like to show that it is possible to save in other file formats as well here. Maybe split into two snippets, first - define your file format, second - save.

Reply:
I really wanted to keep the first part of the tutorial as clean as possible. However, I realize that the flow would be better if some code was co-located. How about two tutorials? One super-basic, and one quite basic? Maybe CaptureTutorial.md and CaptureTutorial_Complete.md / _Full / _Advanced? (maybe customers would find it advanced?)

Reply:
I am a bit sceptical as I don't think we have much to show to have a good reason to have two (which may be good as it shows out sdk is so simple). However, if you havr ideas, let's give it a go.

Reply:
I'm working on two tutorials now. We can later decide to kill the short one.